x-pack/plugins/infra/public/common/alerts/use_summary_time_range.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import {
+  calculateTimeRangeBucketSize,
+  getAlertSummaryTimeRange,
+  useTimeBuckets,
+} from '@kbn/observability-plugin/public';
+import type { TimeRange } from '@kbn/es-query';
+import {
+  DEFAULT_DATE_FORMAT,
+  DEFAULT_INTERVAL,
+} from '../../pages/metrics/hosts/components/tabs/config';
```

Comment:
These 2 are only used in this file. We could move them to `constants` and remove the dependency with `pages`